feature-group-definitions/README.md

```diff
@@ -0,0 +1,82 @@
+# Feature groups
+
+This directory houses the experiment to provide structured definitions groups of web platform features.
+At inception, it consists of this README file only, but it should soon contain data and tools.
+See [ddbeck/common-web-feature-mockup](https://github.com/ddbeck/common-web-feature-mockup/) for more information.
+
+## What is a web platform feature?
+
+To identify, group, and report on the availability of web platform features, it might be nice to have a working defintion of a web platform feature.
+For the purposes of this project, consider this defintion:
+
+**A _feature_ (or _feature group_) is a logical subset of all the addressable behaviors and interfaces that the web platform, as mediated by browsers and related tools, exposes to web developers.**
+
+This definition is surely not authoritative, complete, or universally applicable (see [_Limitations of this definition_](#limitations-of-this-definition)).
+You could come up with many reasonable alternative definitions.
+This definition focuses on things that are useful to this project, but potentially less useful to others.
+
+Also, it's a mouthful. Take a look at each piece:
+
+
+### A logical subset
+
+_A logical subset_ means that all web technologies together are not a feature;
+all of the web platform features taken together **are** the web.
+It also means that a feature does not represent a random or arbitrary selection of web technologies.
+
+For example, a group of JavaScript’s `Symbol` built-in object, the CSS `border-color` property, and the HTTP `POST` method is not a web feature, but all of the CSS border properties (individually or as a group) might be.
+
+
```

Comment:
Since this keeps coming back in discussions, it could be useful to be explicit that features may be at different granularity levels. I suggest the following text but feel free to update or to use some other example.

```suggestion
The definition leaves the size the subset undefined on purpose. Logical subsets may be composed of a single specific feature or hundreds of them. For example, the [range syntax](https://drafts.csswg.org/mediaqueries-4/#mq-ranges) in Media Queries may be a web feature, but so would Media Queries as a whole.
```

Reply:
I like this addition a lot. I think granularity is something implied in some of the other points, but never mentioned explicitly—it's good to highlight it here. 👍